aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLib.cpp

```diff
@@ -956,13 +950,8 @@ void svd_cusolver(const Tensor& A,
 
   static const char* check_svd_doc = "Check doc at https://pytorch.org/docs/stable/generated/torch.linalg.svd.html";
 
-#ifdef ROCM_VERSION
-  // ROCM only has gesvd implemented
-  const auto driver_v = c10::string_view("gesvd");
-#else
   // The default heuristic is to use gesvdj driver
-  const auto driver_v = driver.value_or("gesvdj");
-#endif
+  const auto driver_v = c10::string_view("gesvdj");
```

Comment:
@alugorey Let's retain the CUDA code (`const auto driver_v = driver.value_or("gesvdj");`) as a reference.

Reply:
@jithunnair-amd done!